src/core/dwn-constant.ts

```diff
@@ -3,5 +3,5 @@ export class DwnConstant {
    * The maximum size in bytes of raw data that will be returned as `encodedData`.
    * this is also the maximum size that we will store within MessageStore.
    */
-  public static readonly maxDataSizeAllowedToBeEncoded = 50_000;
+  public static readonly maxDataSizeAllowedToBeEncoded = 42_000;
```

Comment:
the meaning of life 💫

Reply:
I wonder if we should make this configurable in `Dwn.create()`. Made an issue so we can revisit the idea when we have time. #515

Reply:
Consider adding the context info above as comments directly in the code

Reply:
made the number more round with more headroom and added some comments to explain the size choice better.